shared/src/commonMain/kotlin/vn/luongvo/kmm/survey/data/repository/AuthRepositoryImpl.kt

```diff
@@ -32,8 +31,17 @@ class AuthRepositoryImpl(
         tokenLocalDataSource.saveToken(token)
     }
 
+    override fun clearToken() {
```

Comment:
@luongvo Optional, to avoid confusion, I think it would be better if we name it `clearLocalToken`.

Reply:
@doannimble Yes, I fixed it in the next PR https://github.com/luongvo/kmm-survey/pull/72/files#diff-50e2a0690a008961d0d9a746b1ed14a393541fe5d720c3dd455b6b772ea4c3aaR34 ✅